src/mudlet.pro

```diff
@@ -519,7 +522,8 @@ SOURCES += \
     TVar.cpp \
     VarUnit.cpp \
     XMLexport.cpp \
-    XMLimport.cpp
+    XMLimport.cpp \
+    ../3rdparty/pugixml/src/pugixml.cpp
```

Comment:
Qt Creator always appends to the list but we may want to keep it sorted (case insensitively) for ease of finding items.

Reply:
Sorted it for the time being!